controllers/operatorconfig_controller.go

```diff
@@ -54,51 +78,315 @@ type OperatorConfigReconciler struct {
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.10.0/pkg/reconcile
 func (r *OperatorConfigReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	log := log.FromContext(ctx)
+	// Fetch the OperatorConfig operatorConfig
+	operatorConfig := &cranev1alpha1.OperatorConfig{}
+	if err := r.Get(ctx, types.NamespacedName{
+		Namespace: InstallNamespace,
+		Name:      OwnerConfigName,
```

Comment:
This should be `req.Namespace` and `req.Name` right?

Reply:
We don't want to reconcile for multiple `operatorConfig` CR so restricting reconcile loop to the only CR we care about.

Reply:
True, but what if the user created CR is not named `openshift-migration`? the get here will keep failing

Reply:
We mention that in the log, that desired `operatorConfig` is not found and we end the reconcile loop. we don't reconcile again till new object is created

Reply:
> We mention that in the log, that desired operatorConfig is not found and we end the reconcile loop. we don't reconcile again till new object is created

This is not a great UX right? a user should not have to look into logs for something as easy as this.

Reply:
Adding something in the staus field would make sense to better UX? @alaypatel07 @djzager

Reply:
Yeah. Just so we are clear on what needs to happen:
1. We need to reconcile the request we receive. We cannot just take any event on the queue and reconcile the one we care about.
2. If the user creates one in a different namespace or with a different name...we simply reject it (we can put it on the `.status` that would be a good experience) -- I'm okay with this being fixed in a follow-up PR (please create an issue if you are going this route).

Reply:
For now I added `message` in status of `operatorConfig` but I will file an issue to plan some better way to present errors and warnings